packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_home/views/home_page.dart

```diff
@@ -78,6 +78,14 @@ class HomePage extends StatelessWidget {
           routePath: RoutesPath.widgetToolkit,
         ),
         {{/enable_feature_widget_toolkit}}
+        {{#enable_feature_qr_scanner}}
+            NavMenuItem(
+          title: context.l10n.featureQr.qrCodePageTitle,
+          icon: context.designSystem.icons.qrCode,
+          route: const QrCodeRoute(),
+          routePath: RoutesPath.qrCode,
+        ),
+        {{/enable_feature_qr_scanner}}
```

Comment:
**No need to change anything in this PR**, just adding this comment for further reference: 

We should really consider adding a showcases navbar and replacing other showcase nav bars containing those, as currently there are 7 navbars in projects generated with all flags enabled (making things really cramped and on smaller screens harder to pick).

While iOS doesn't explicitly require a number of navbars present, the [newest material design](https://m3.material.io/components/navigation-bar/overview) suggests adding 3-5 items of equal importance.